doc/reference/storages.rst

```diff
@@ -144,31 +144,46 @@ Options:
 Noteworthy non-included storages
 ================================
 
-A number of important ZODB storages are distributed separately, including:
+A number of important ZODB storages are distributed separately.
+
+Base storages
+-------------
+
+Unlike the included storages, all the implementations listed in this section
+allow multiple processes to share the same database.
+
+NEO
+  `NEO <https://lab.nexedi.com/nexedi/neoppod>`_ can spread data among several
+  computers for load-balancing and multi-master replication. It also supports
+  asynchronous replication to off-site NEO databases for further disaster
+  resistance without affecting local operation latency.
```

Comment:
I would consider it important to mention that of all the storages mentioned here, NEO is the only one that's Linux only; all the rest of them are operating-system independent. (The linux-only requirement is the major reason my organization can't test NEO---we develop on OS X.)